### ISSUE TITLE
input types

### ISSUE THREAD

**Issue by Thebigbignooby** [mhallin/graphql-docs#10]
Any reason as to why input types are not documented ?

**Comment by cmosboss** [comment]
I just ran into this, sucks that it is not addressed :( Gotta find another package I guess.

**Comment by GlennOlsson** [comment]
I'm pretty late to the ball here but I guess this could help the people in the future with the same problem. Instead of creating a PR to fix this as the repo seems dead, I'll just post the solution I came up with.

Debugged trough the compiled and unreadable React document which was a bitch, took about 1.5 hours. Finally found out that if the input type does not have any description, it will not render (which mine didn't) as seen [here](https://github.com/mhallin/graphql-docs/blob/f06bdf2aa08e077a3f3b538f2bd44a20d77115ef/src/components/FieldArgumentsTableView.js#L18). Just removing this filtering will render the types (at least for me). This can be solved by, in the generated HTML document, replace `this.props.args.filter(function(e){return e.description})` with just `this.props.args`

**TL;DR;** Replace the one occurrence of `this.props.args.filter(function(e){return e.description})` with `this.props.args` (just search and replace)
